x-pack/plugins/alerting/server/data/rule/types/rule_attributes.ts

```diff
@@ -125,7 +125,7 @@ interface AlertsFilterAttributes {
 
 export interface RuleActionAttributes {
```

Comment:
Why we did not create here two types(system and default) instead of one?

Reply:
Because this type represents what is being saved in ES. The data in ES should not have the action's `type`.